4/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN set -eux; \
     groupadd --system --gid 1000 scanner-cli; \
     useradd --system --uid 1000 --gid scanner-cli scanner-cli; \
     apt-get update; \
-    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck nodejs>12 build-essential; \
+    apt-get install -y --no-install-recommends gnupg unzip wget bash fonts-dejavu python3 python3-pip shellcheck nodejs>14 build-essential; \
```

Comment:
Neither `nodejs>12` or `nodejs>14` is valid syntax for `apt-get`. This syntax works in Alpine which was previously used before a195c3c5c041dfb433381b444b0dc42eb27a0ab7

The new image uses ubuntu jammy which only has v12 available.

Reply:
The new image needs to setup the nodesource repository for the current node LTS version before calling `apt-get`.

Reply:
Suggested changes have been incorporated to the PR that should resolve this comment :)

Reply:
And the image is building fine? Because I've not tested them.

Reply:
I've not tested this, but `curl` should be already installed by the base image ( https://github.com/adoptium/containers/blob/main/17/jre/ubuntu/jammy/Dockerfile.releases.full#L29 )

Reply:
To be safe and futureproof, I think curl shall be installed before calling the nodesource script: 

```shell
apt-get install -y --no-install-recommends ca-certificates curl bash
curl -fsSL https://deb.nodesource.com/setup_lts.x | bash
...
apt-get install -y --no-install-recommends gnupg unzip wget fonts-dejavu python3 python3-pip shellcheck nodejs build-essential
```